### PR TITLE
[artifactory-ha] Add custom labels in Nginx service

### LINKED ISSUE #474: [artifactory-ha] - Add custom labels in Nginx service
**Is this a BUG REPORT or FEATURE REQUEST?**: FEATURE REQUEST

**Which chart**: artifactory-ha

**What you expected to happen**: To have the chance to define custom labels for the nginx service

**Anything else we need to know**: Here a feature with the suggestion: https://github.com/guillemtanya/charts/tree/feature/add_nginx_custom_labels

### PR DESCRIPTION
**Pull request by guillemtanya**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:  To have the chance to define custom labels for the nginx service

**Which issue this PR fixes**: fixes #474

@amithins
@danielezer 
@eldada 
@rimusz

### CODE REVIEW

File: stable/artifactory-ha/README.md [hunk 1]
```diff
@@ -811,6 +811,7 @@ The following table lists the configurable parameters of the artifactory chart a
 | `nginx.service.type`        | Nginx service type                | `LoadBalancer`                                         |
 | `nginx.service.clusterIP`   | Specific cluster IP or `None` for headless services     | `nil`                         |
 | `nginx.service.loadBalancerSourceRanges`| Nginx service array of IP CIDR ranges to whitelist (only when service type is LoadBalancer) |        |
+| `nginx.service.labels`       | Nginx servicelabels     | `{}`                                                           |
```
Comment (danielezer) [inline stable/artifactory-ha/README.md]: You are missing a space in servicelabels

**Comment by danielezer** [comment]
Please bump chart version and add a line to the changelog

File: stable/artifactory-ha/README.md (hunk 1 above)
Comment (guillemtanya) [reply stable/artifactory-ha/README.md]: @danielezer fixed

**Comment by guillemtanya** [comment]
> Please bump chart version and add a line to the changelog

Bumped and updated @danielezer

**Comment by eldada** [comment]
Please fix conflicts

**Comment by eldada** [comment]
Please fix conflicts

**Comment by guillemtanya** [comment]
> Please fix conflicts

Fixed! And version & Changelog updated

**Comment by eldada** [comment]
Can you also add this to `artifactory` chart?

**Pull request merged**